zzz_generated/app-common/app-common/zap-generated/ids/Commands.h

```diff
@@ -34,14 +34,14 @@ namespace Identify {
 static constexpr CommandId Id = 0x00000000;
 } // namespace Identify
 
-namespace IdentifyQueryResponse {
```

Comment:
I assume this is because the new `zcl_commands` impl returns things in a different order?  

Are the ZAP repo changes needed if we don't need `command_arg_contains_nested_array` (as I suspect we don't)?

Reply:
Fixed,thanks